tensorboard/components/tf_runs_selector/tf-wbr-string.html

```diff
@@ -0,0 +1,65 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../tf-imports/polymer.html" />
+
+<!--
+  tf-wbr-string safely renders a string, with <wbr> elements inserted
+  around select delimiters (see `const delimiterPattern` in this file).
+-->
+<dom-module id="tf-wbr-string">
+  <template>
+    <!--
+      This ugly formatting is required to prevent spaces from slipping
+      into the HTML.
+    -->
+    <template is="dom-repeat" items="[[_parts]]" as="part"
+      >[[part]]<wbr
+    /></template>
+  </template>
+  <script>
+    Polymer({
+      is: 'tf-wbr-string',
+      properties: {
+        /** The value to render with word breaks. */
+        value: String,
+        _parts: {
+          type: Array /* string[] */,
+          computed: '_computeParts(value)',
+        },
+      },
+      _computeParts(value) {
+        const result = [];
+        const delimiterPattern = /[/=_,-]/;
+        if (value == null) {
+          value = '';
+        }
+        while (true) {
+          const idx = value.search(delimiterPattern);
+          if (idx === -1) {
+            result.push(value);
```

Comment:
Stating obvious: the `value` can be an empty string and you are pushing an empty string into the `result` which can cause multiple `<wbr><wbr>` which probably has zero visual impact.

Reply:
Yep!